yaml_edit/lib/src/utils.dart

```diff
@@ -78,9 +74,7 @@ bool isBlockNode(YamlNode node) {
 
 /// Returns the content sensitive ending offset of [yamlNode] (i.e. where the last
 /// meaningful content happens)
-int getContentSensitiveEnd(YamlNode yamlNode) {
-  ArgumentError.checkNotNull(yamlNode, 'yamlNode');
-
+int getContentSensitiveEnd(YamlNode? yamlNode) {
```

Comment:
When can `yamlNode` be `null`? I'm guessing it can't, and we should push the null check into all callers.

Reply:
Yeah, I don't remember why I did this. :(

Reply:
Now I think I remember. On `_replaceInBlockMap()` was a call to `getContentSensitiveEnd(map.nodes[key]);`. I've since pushed the null assertion there.